README.md

```diff
@@ -231,11 +231,13 @@ Full C/C++ API is described at the top of the `webview.h` file.
 13. `webview_inject_css()` has been removed. Use `webview_eval()` to create style tag manually.
 14. `webview_debug()` has been removed. Use whatever fits best to your programming language and environment to debug your GUI apps.
 
-## Notes
+## Known issues
```

Comment:
I think we should keep it "Notes", since some of the items listed aren't strictly issues. Otherwise, LGTM!